### DISCUSSION TITLE
Downloads Pane/Section

### DISCUSSION THREAD

**Discussion by Snake883** [Tribler/tribler#5803] [Ideas]
##### Actual behavior: #####
I think the section in the left pane that says "Downloads", and the main tab that says "Downloads" should be changed to something else.

![image](https://user-images.githubusercontent.com/8312560/101196806-1f4c1080-3659-11eb-85ef-338d61083e55.png)

##### Expected behavior: #####
Why?
Because there's more activity going on than just downloading.
I'm seeding.
I'm also passing/contributing data to the Tribler network (which would be nice to see in near real time so I know what's going on with my own network).

I think the pane titles should be changed from "Downloads" to "Activity" or "Activities".

Also, I think the headers "ALL", "DOWNLOADING", "COMPLETED", "ACTIVE", "INACTIVE" should match and be filters for the torrent STATUS.

##### Tribler version/branch+revision: #####
v7.6.0-exp1

##### Operating system and version: #####
Windows

**Comment by ichorid** [comment]
Thanks for the suggestion! We already changed a lot in the GUI for this release. Also, we're still a torrent client after all, and torrent clients tend to have the "Downloads" section :wink:. So we will keep this for awhile.

Though, I agree that at least _something_ should be done with the downloads section appearance.

**Comment by Snake883** [comment]
> torrent clients tend to have the "Downloads" section 

I think Tribler already has a "Downloading" section in the right pane.  Today, even the left side "Downloads" section is not just download activity.

I also want to see Tribler network activity tab, because I need to manage my bandwidth (fixed monthly allotment, working from home, etc).

I also propose Tribler to encourage community contributions (seeders, Tribler network) within the GUI.  The more encouragement and features to support contributing members, the less likely we'll see dead torrents.

![image](https://user-images.githubusercontent.com/8312560/101201076-165e3d80-365f-11eb-9fa5-300261da7ec3.png)
